components/Filters.js

```diff
@@ -83,10 +91,11 @@ const Option = withStyles(theme => ({
     padding: '4px 10px',
     margin: '4px 2px',
     cursor: 'pointer',
+    border: ({ chip, selected }) =>
+      chip || selected ? `1px solid ${theme.palette.secondary[100]}` : 'none',
```

Comment:
Selected categories should also have borders

Figma:
![image](https://user-images.githubusercontent.com/108608/85227090-aee46680-b40d-11ea-9130-8aa5036b3514.png)


Now:
![image](https://user-images.githubusercontent.com/108608/85227083-9ffdb400-b40d-11ea-87d5-e84e83cca4c2.png)